### PR TITLE
Routing to different collections based on origin ids.

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -89,23 +95,24 @@ func main() {
...
 		srcConf := consumer.QueueConfig{
 			Addrs:                *sourceAddresses,
 			Group:                *sourceGroup,
 			Topic:                *sourceTopic,
 			Queue:                *sourceQueue,
 			ConcurrentProcessing: *sourceConcurrentProcessing,
 		}
-
+		initLogs(os.Stdout, os.Stdout, os.Stderr)
+		var collectionsByOriginIds []CollectionByOriginId
```
Comment (tamas-molnar) [inline app.go]: Maybe make this a map to make lookup easier?

File: app.go [hunk 2]
```diff
@@ -152,50 +159,60 @@ func readMessages(config consumer.QueueConfig) {
...
-
 	response, err := client.Do(request)
-
 	if err != nil {
 		errorLogger.Printf("[%s] Error caling writer at [%s]: [%v]", tid, requestURL, err.Error())
 		return
 	}
-	defer response.Body.Close()
+	defer func() {
+		io.Copy(ioutil.Discard, response.Body)
```
Comment (Nesze) [inline app.go]: I think this is already covered by line 202: 
https://github.com/Financial-Times/native-ingester/pull/3/files#diff-a66cc59eb6bfe62b7b3b69cd52b84747R202.

So probably remove one of them. Probably line 202.

File: app.go (hunk 1 above)
Comment (Nesze) [inline app.go]: +1

File: app.go (hunk 2 above)
Comment (kovacshuni) [inline app.go]: fixed. at all occureneces.

File: app.go (hunk 1 above)
Comment (kovacshuni) [inline app.go]: ok

**Comment by Nesze** [comment]
+1

**Comment by tamas-molnar** [comment]
+1

**Pull request merged**